lib/libtlspool.c

```diff
@@ -487,11 +305,7 @@ static void *master_thread (void *path) {
 			//
 			// Wait before repeating, with exponential back-off
 			if (poolfd == INVALID_POOL_HANDLE) {
-#ifdef WINDOWS_PORT
-				Sleep(usec / 1000);
-#else
-				usleep(usec);
-#endif
+				os_sleep(usec);
 				usec <<= 1;
```

Comment:
ok, dit is dus iets wat ik als auditor zou kunnen verkroppen.  ik hoef maar 1x mijn vermoeden zeker te stellen dat dit precies doet wat ik verwacht.  overigens is os_usleep dan nog helderder als naam.